scripts/quota_log.py

```diff
@@ -35,12 +47,19 @@ def disk_quota():
 
 def cpu_hours():
     current_time = datetime.datetime.now()
-
-    # script that runs on uppmax
-    uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    try:
+        # script that runs on uppmax
+        uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    except Exception, e:
+        logging.error(e.message)
+        exit()
```

Comment:
Didn't actually know about Python exit()!

The problem with it is that it is very imperialistic to do exit() like this. It really is dictatorship over anyone trying to use this code as a module. Much better to raise an exception for the user to deal with as he/she sees fit.

os.exit(1) is slightly better since it sends an error state to the environment, as you say.